app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' && controller_name != 'registrations' %>
+  <%= link_to "Log in", new_session_path(resource_name), class: "text-decoration-none mb-1", style: "color: #989da3" %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' && controller_name != 'sessions' %>
+  <%= link_to "Sign up", new_registration_path(resource_name), class: "text-decoration-none mb-1", style: "color: #989da3" %><br />
+<% end %>
+
+<%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
+  <%= link_to "Forgot password ?", new_password_path(resource_name), class: "text-decoration-none mb-1", style: "color: #989da3" %><br />
+<% end %>
+
+<%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
+  <%= link_to "Didn't receive confirmation instructions?", new_confirmation_path(resource_name), class: "text-decoration-none", style: "color: #989da3" %><br />
+<% end %>
+
+<%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
+  <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name), class: "text-decoration-none", style: "color: #989da3" %><br />
+<% end %>
```

Comment:
**Suggested Changes**
- [ ] When I have signed in, the Register and login links shouldn't be visible as shown below.
![menu](https://user-images.githubusercontent.com/7220167/143585183-eb0c4572-bdc5-4de9-abf7-912bc41a5cf7.png)